pkg/strategy/grid2/active_order_recover.go

```diff
@@ -125,9 +126,13 @@ func syncActiveOrders(ctx context.Context, opts SyncActiveOrdersOpts) error {
 			// sleep 100ms to avoid DDOS
 			time.Sleep(100 * time.Millisecond)
 
-			if err := syncActiveOrder(ctx, opts.activeOrderBook, opts.orderQueryService, activeOrder.OrderID); err != nil {
-				opts.logger.WithError(err).Errorf("[ActiveOrderRecover] unable to query order #%d", activeOrder.OrderID)
-				errs = multierr.Append(errs, err)
+			if err := syncActiveOrder(ctx, opts.activeOrderBook, opts.orderQueryService, activeOrder.OrderID, syncBefore); err != nil {
+				if strings.Contains(err.Error(), "skip syncing active order") {
+					opts.logger.Infof("[ActiveOrderRecover] skip syncing active order #%d, because the updated_at is in 3 min", activeOrder.OrderID)
```

Comment:
but why returning error for skipped order?

Reply:
maybe return bool to let upper function know what happened ?